packages/matchbox/src/components/Drawer/styles.js

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components';
+import { tokens } from '@sparkpost/design-tokens';
+
+export const Overlay = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
```

Comment:
Could use Box here and set these styles on the component instead of in the styles, but honestly fine either way